src/main/java/it/smartcommunitylab/aac/config/DatabaseConfig.java

```diff
@@ -177,24 +192,24 @@ public JpaTransactionManager getTransactionManager(EntityManagerFactory entityMa
     @Bean(name = "coreJdbcDataSourceInitializer")
     public JdbcDataSourceInitializer jdbcDataSourceInitializer(
         @Qualifier("jdbcDataSource") DataSource dataSource,
-        JdbcProperties properties
+        @Qualifier("jdbcProperties") JdbcProperties properties
     ) {
         return new JdbcDataSourceInitializer(dataSource, properties);
     }
 
     @Bean(name = "oauth2JdbcDataSourceInitializer")
     public JdbcDataSourceInitializer oauth2DataSourceInitializer(
         @Qualifier("jdbcDataSource") DataSource dataSource,
-        JdbcProperties properties
+        @Qualifier("jdbcProperties") JdbcProperties properties
     ) {
         return new JdbcDataSourceInitializer(dataSource, properties, "classpath:db/sql/oauth2/schema-@@platform@@.sql");
     }
 
     //TODO add optional ObjectProvider<DataSource> to support separated dataSource for audit
```

Comment:
IMO either fulfill the TODO with the commit or remove the TODO. In the current state, there is a TODO for a functionality that already exists.